integration_tests/fees_along_path.feature

```diff
@@ -0,0 +1,13 @@
+Feature: Fees along path
+
+    Scenario:
+        Given we have a scenario named grid_fees/test_grid_fees
+        And d3a is installed
+        When we run the simulation with setup file grid_fees.test_grid_fees and parameters [24, 60, 60, 1]
+        Then the load initial buying rate adds grid fees
+
+    Scenario: 
+        Given we have a scenario named grid_fees/test_grid_fees
+        And d3a is installed
+        When we run the simulation with setup file grid_fees.test_grid_fees and parameters [24, 60, 60, 1]
+        Then the PV initial selling rate subtracts grid fees
```

Comment:
I think these two scenarios could be squashed into one simply via adding the step `the PV initial selling rate subtracts grid fees` after L7 at this point of commit

Reply:
Would it start with "And" or "Then"?

Reply:
All the following conditions would start with `And`